File: PJDesignBackStageClient/src/app/features/type2/feature-shared/models/create-or-update-type2-content.ts
import { EditRequestBase } from "src/app/shared/models/bases";

export class CreateOrUpdateType2ContentRequest extends EditRequestBase {
  id?: number | null;
  unitId: number;
  title: string;
  description: string;
  isEnabled: boolean;
  isFixed: boolean;
  content: string;
  categoryIDs: number[] | null | undefined;
  thumbnailUrl: string;
  imageUrl: string;
}

File: PJDesignBackStageAPI/App.Model/Type2/CreateOrUpdateType2Content.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace App.Model
{
    public class CreateOrUpdateType2ContentRequest : EditRequestBase
    {
        public int? Id { get; set; }
        public int UnitId { get; set; }
        public bool IsEnabled { get; set; }
        public bool IsFixed { get; set; }
        public string Title { get; set; } = null!;
        public string Description { get; set; } = null!;
        public string Content { get; set; } = null!;
        public string ThumbnailUrl { get; set; } = null!;
        public string ImageUrl { get; set; } = null!;
        public IEnumerable<int>? CategoryIDs { get; set; }
    }
}
